### DISCUSSION TITLE
Upgrade from minimum fails repeatedly

### DISCUSSION THREAD

**Discussion by jds11111** [arendst/Tasmota#16965] [Q&A]
### PROBLEM DESCRIPTION
I have an ESP8266 power plug that I was updating to the latest Tasmota release 12.2.0. It upgrades to the minimal version, but not to tasmota.bin. I tried many times through the Tasmota releases web page, and also by uploading the bin file from my computer, both zipped and not zipped. The plug disappears from network activity for several minutes, and always eventually reappears with minimal tasmota running. 

When it finally reappears, this info is in the console:
{"Info3":{"RestartReason":{"Exception":28,"Reason":"Blocked Loop","EPC":["402015be","00000000","00000000"],"EXCVADDR":"00000000","DEPC":"00000000","CallChain":["402015bc","40239e4c","4021bd72","402405f0","4021c442","402405fd","40240642","40219884","40000f49","40000f49","40000f49","40000e19","40104e01","40104e07","4010000d","4023906c","4023901d","40104c49","4010482e","4010417d","40104731","40104a09","40102cd2","4000050c","401029a4","4021e5ea","4021e5ed","40100214","401055a1","4000050c","4021eeac"]},"BootCount":30}}

I am beginning to suspect a hardware failure, but do not know how to test that.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x ] Device used (e.g., Sonoff Basic): GDTech MPV2RO-US Plug
- [ x] Tasmota binary firmware version number used: __12.2.0___
  - [ x] Pre-compiled
  - [ ] Self-compiled
- [x ] Flashing tools used: _OTA____
- [x ] Provide the output of command: `Backlog Template; Module; GPIO 255`: 
```lua
  Configuration output here:
Command unknown
```lua

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
command unknown

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by barbudor** [comment]
> It upgrades to the minimal version, but not to tasmota.bin

You should be uploading tasmota.bin.gz
Always use the compressed binary for OTA
http://ota.tasmota.com/tasmota/release/tasmota.bin.gz

**Reply by jds11111** [reply]
Yeah, I know, I just tried the unzipped to see how it complained.

**Comment by Jason2866** [comment]
Since you did not provide all relevant infos your question can not be answered.

**Reply by jds11111** [reply]
What is missing?

**Reply by Jason2866** [reply]
everything asked in the template

**Reply by jds11111** [reply]
OK, I give up then.  I have tried everything that I can see above, and put whatever information is provided, or it does nothing. 
Anyway, I suspect that it is something with hardware.

**Reply by barbudor** [reply]
Any chance you open the plug and serial flash?
You haven't tell any details about the device itself

**Reply by jds11111** [reply]
It is the plug mentioned in the post (https://templates.blakadder.com/gdtech_MPV2RO-US.html). These seem to be a pain to open, and then probably don't really ever go back together right, so seems not worth it.

**Reply by barbudor** [reply]
Ok

**Reply by michaelvvvvv** [reply]
I have the exact same problem with one off my 8 NOUS A1T plugs.
